modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-dispatch-task-executor/src/main/java/com/liferay/site/initializer/testray/dispatch/task/executor/internal/dispatch/executor/util/ObjectEntryUtil.java

```diff
@@ -86,7 +87,10 @@ public static ObjectEntry getObjectEntry(
 			ObjectEntryManager objectEntryManager)
 		throws Exception {
 
-		return objectEntryManager.getObjectEntry(
+		DefaultObjectEntryManager defaultObjectEntryManager =
```

Comment:
In this case it's not needed because the instance always will be an instance of DefaultObjectEntryManager,

they are referencing this way below:

````java
@Reference(target = "(object.entry.manager.storage.type=default)")
private ObjectEntryManager _objectEntryManager;
````